examples/get_namespaced_configmap.py

```diff
@@ -0,0 +1,36 @@
+from kubernetes import client, config
+from kubernetes.client.rest import ApiException
+
+
+def get_configmap_from_namespace(api_token_dir: str, accessing_kubernetes_api_url: str, namespace_name: str):
+    """
+    :param api_token_dir: directory to your ApiToken
+        Note: to get ApiToken use bash command below:
+            kubectl describe secret $(kubectl get secrets | grep default | cut -f1 -d ' ') | grep -E '^token' | cut -f2 -d':' | tr -d '\t'
+    :type api_token_dir: str
+    :param accessing_kubernetes_api_url: url that provides access to kubernetes api
+        Note: to access kubernetes api use command below
+            kubectl proxy --port=8080
+            result will be like: Starting to serve on 127.0.0.1:8080
+    :type accessing_kubernetes_api_url: str
+    :param namespace_name: namespace name that contains configmap
+    :type namespace_name: str
+    :return: True if response received successfully, False otherwise
+    """
+    configuration = client.Configuration()
```

Comment:
As an example it is ok to instantiate dependencies locally, but in future please provide them by argument - this will help you test your function